Gemfile.lock

```diff
@@ -47,7 +47,7 @@ GEM
       multi_json (~> 1.0)
     addressable (2.3.5)
     arel (3.0.3)
-    bcrypt (3.1.7)
+    bcrypt (3.1.10)
```

Comment:
Ok I've done a lot more thinking/investigation about `Gemfile`/`Gemfile.lock`/`slices.gemspec` since I wrote the above, and how we use `Gemfile.lock` has changed. Now we delete it in CI, we always get all the latest possible versions of dependencies, so that means any changes to the _committed_ `Gemfile.lock` don't affect our confidence in Slices working with those dependencies. So that's nice.

So the `Gemfile.lock` ends up being a "reference" set of versions, which we know pass the tests (see https://github.com/rails/rails/pull/18992). So then the question remains: do we want downstream dependencies of the gem we're trying to update also be updated? I think the answer is still no - we should remain conservative with our updates and only update the gem we're interested in (unless, of course, that gem _requires_ its dependencies to be updated). It keeps things simpler.